services/organization.js

```diff
@@ -1,14 +1,21 @@
+const {toISO} = require("../crdc-datahub-database-drivers/utility/time-utility");
 class Organization {
     constructor(organizationCollection) {
         this.organizationCollection = organizationCollection;
     }
 
     async getOrganizationByID(id) {
         let result = await this.organizationCollection.find(id);
-        return result?.length > 0 ? result[0] : null;
+        return result?.length > 0 ? toISOTime(result[0]) : null;
     }
 }
 
+const toISOTime = (aOrg) => {
```

Comment:
This function name is a bit misleading, people may think we are converting the object into an ISO time. I suggest rename it to something like transformDateTime, transformDateTimeFormat or similar.

Reply:
updated!